nebula-logger/main/log-management/classes/LogHandler.cls

```diff
@@ -59,34 +58,6 @@ public without sharing class LogHandler extends LoggerSObjectHandler {
         this.shareLogsWithLoggingUsers();
     }
 
-    private void setOrgReleaseCycle() {
-        // Currently, there doesn't seem to be a way within Apex or api.status.salesforce.com...
```

Comment:
yeah. I think removing this is a really good change. I was never crazy about you having to hardcode the pre-release instances!

Reply:
Yeah, I really wanted to make it work, but it's been tedious keeping it up to date (even though it's only 3 releases/year). If/when I find a way to automatically determine it, I'll definitely consider re-adding the field.